modules/cnv_note_imports.adoc

```diff
@@ -0,0 +1,9 @@
+[[note-on-imports]]
+=== A note on importing disk images
```

Comment:
I think this module should be a step in the disk image importing procedure, and, if guidance is available for when you need to resize the file system, you should provide it. If you always need to resize the file system, you should just say to do it.

Reply:
Well it doesn't NEED to be resized, but since the disk image got resized, the partitions for whatever guest os you are running should be resized to match so you don't have a bunch of unused virtual disk space.

Reply:
@kalexand-rh It's no longer a separate module, but I included it as a warning in the two importing procedures. How does that look?